downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md

```diff
@@ -6,29 +6,39 @@ title: Swan Lake Preview 1 Release note
 This release is the first preview version of Ballerina Swan Lake. This release includes a new set of Language features and significant improvements to the compiler, runtime, standard libraries, and developer tooling.
  
 # Highlights
-- Highlight I
-- Highlight II
+
+- New recursive descent parser
+- Type system enhancements
+- Improved language integration support
+- Improvements on Standard Library modules 
+- Improved mocking support in testing
 
 # What's new in Ballerina Swan Lake - Preview 1
+
 ## Language
+
 The Language implementation is based on [Ballerina Language Specifications Draft 2020-06-18](https://ballerina.io/spec/lang/draft/v2020-06-18/). This Specification introduces a set of new features and improvements in the following main areas.   
  
-- Type system enhancements (e.g. New Error and Table Design) 
-- Improved Immutability support 
-- [New Transaction support] (https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/proposals/transaction/transaction.md) 
-- Improved Query support 
+- Type system enhancements (e.g., new error and table design) 
+- Improved immutability support 
+- [New transactions support](https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/proposals/transaction/transaction.md) 
+- Improved query support 
  
-Some of these new language features are revamped of the existing language features and Therefore source code will not be backward compatible with the stable Ballerina 1.2 releases. 
+Some of these new language features are revamped versions of the existing language features. Therefore, the source code will not be backward compatible with the stable Ballerina 1.2 releases. 
  
-In addition to the new Language features, this release introduces a new parser implementation aiming to improve the performance and usability of the compiler. Now compiler has more control over syntax errors, and it can provide a better diagnostic for syntax errors. Additionally, New parser has tightened up the language parser rules with respect to the Ballerina specification, and still, it doesn’t cover the full set of the language features. This will be fixed in upcoming preview versions.  
+In addition to the new Language features, this release introduces a new parser implementation aiming to improve the performance and usability of the compiler. Now, the compiler has more control over syntax errors and it can provide a better diagnostic for syntax errors. Additionally, the new parser has tightened up the language parser rules with respect to the Ballerina specification. However, still it doesn’t cover the full set of the language features. This will be fixed in the upcoming preview versions.  
```

Comment:
```suggestion
In addition to the new language features, this release introduces a new parser implementation aiming to improve the performance and usability of the compiler. Now, the compiler has more control over syntax errors and it can provide a better diagnostic for syntax errors. Additionally, the new parser has tightened up the language parser rules with respect to the Ballerina specification. However, it still doesn’t cover the full set of the language features. This will be fixed in the upcoming preview versions.  
```